node/node.go

```diff
@@ -356,6 +356,7 @@ func (n *Node) obtainJWTSecret(cliParam string) ([]byte, error) {
 	if data, err := os.ReadFile(fileName); err == nil {
 		jwtSecret := common.FromHex(strings.TrimSpace(string(data)))
 		if len(jwtSecret) == 32 {
+			log.Info("Read JWT secret", "path", fileName)
```

Comment:
One thing I've been thinking of, is if we should output even more info. Like, perhaps the crc32 checksum of the secret? It's annoying trying to figure out of EL and CL have the _same_ secret. 
Would be easier if they both spit out some low-entropy signature.. (?)

Reply:
Like, how about something like this? 
```golang
			log.Info("Read JWT secret", "path", fileName, "crc32", fmt.Sprintf("%#x", crc32.ChecksumIEEE(jwtSecret)))
```

Reply:
```
INFO [06-15|10:17:36.628] Read JWT secret                          path=/media/ropsten/geth/jwtsecret crc32=0x4ce4e295
```